bin.src/add_fbs_proposals.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+import matplotlib
+matplotlib.use('Agg')
+
+import argparse
+import sqlite3
+from sqlite3 import OperationalError, IntegrityError
+
+import numpy as np
+import pandas as pd
+
+import healpy as hp
+from lsst.sims.featureScheduler import utils as schedUtils
+
+__all__ = ['get_standard_wfd', 'get_extended_wfd', 'get_extended_wfd_dust',
+           'define_ddname', 'get_visits', 'label_visits', 'update_database']
+
+
+def get_standard_wfd(nside=64):
+    wfd_footprint = schedUtils.WFD_no_gp_healpixels(nside, dec_min=-62.5, dec_max=3.6,
+                                                    center_width=10., end_width=4.,
+                                                    gal_long1=290., gal_long2=70.)
+    return wfd_footprint
+
+
+def get_extended_wfd(nside=64):
+    wfd_footprint = schedUtils.WFD_no_gp_healpixels(nside, dec_min=-72.25, dec_max=12.4,
+                                                    center_width=14.9,
+                                                    gal_long1=0, gal_long2=360)
+    return wfd_footprint
+
+
+def get_extended_wfd_dust(nside=64):
```

Comment:
I'm a little worried that these `get_*_wfd` functions have a lot of magic numbers that could get out of sync with the scheduler repo. Do we really need to wrap one-line calls in utility functions? 

It probably means it's finally time to make a footprint class rather than just using dicts. Then the footprint object can have extra info about where WFD is for convenience and the scheduler can just ignore those bits.